### ISSUE TITLE
will there be future GeoMx blog posts on how to analyze the data?

### ISSUE THREAD

**Issue by acjordan333** [Nanostring-Biostats/CosMx-Analysis-Scratch-Space#129]
Hi Bruker/Nanostring team, 

Thanks for maintaining the CosMx scratch space blog, which is excellent for those of us who are conducting all our analyses in the command line outside of AtoMx. Are there plans to create something similar for GeoMx (a GeoMx scratch space blog)? The Nov 2021 vignette is great, but I was wondering if there would be updates on topics pertaining to GeoMx such as how to correct for batch effect, best practices for using scoring systems such as GSVA/ssGSEA, detecting spatial patterns in tissues, any updates on getting the best deconvolution results if you don't have matching scRNA data for a study, etc. 

Best,
Alex

**Comment by eveilyeverafter** [comment]
Yes, there are plans to do posts with GeoMx DSP. We'll likely broaden the scope of Scratch Space to include DSP instead of creating a separate blog.

**Comment by acjordan333** [comment]
Excellent! Thanks for responding so quickly. What's the ETA for additional GeoMx blog posts?

**Comment by eveilyeverafter** [comment]
Forgive me for missing your response. I have one GeoMx post that was update last summer (https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/posts/GeoMx-scMask-generation/). If you have any post that you would like to see, please create a new issue.
